nixos/modules/programs/yubikey-manager.nix

```
@@ -0,0 +1,25 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.yubikey-manager;
+in
+{
+  options = {
+    programs.yubikey-manager = {
+      enable = lib.mkEnableOption "yubikey-manager";
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
+
+    # The udev rules we want aren't included in the yubikey-manager package, but
+    # we can get them from yubikey-personalization.
+    services.udev.packages = [ pkgs.yubikey-personalization ];
```

```suggestion
    services = {
      pcscd.enable = true;

      # The udev rules we want aren't included in the yubikey-manager package, but
      # we can get them from yubikey-personalization.
      udev.packages = [ pkgs.yubikey-personalization ];
    };
```

Got this from my config, without not all functions in yubikey-managher function.